optaplanner-examples/src/main/resources/org/optaplanner/examples/vehiclerouting/vehicleRoutingSolverConfig.xml

```diff
@@ -7,7 +7,7 @@
   <!--<moveThreadCount>AUTO</moveThreadCount>-->
 
   <solutionClass>org.optaplanner.examples.vehiclerouting.domain.VehicleRoutingSolution</solutionClass>
-  <entityClass>org.optaplanner.examples.vehiclerouting.domain.Standstill</entityClass>
+  <entityClass>org.optaplanner.examples.vehiclerouting.domain.Vehicle</entityClass>
   <entityClass>org.optaplanner.examples.vehiclerouting.domain.Customer</entityClass>
   <entityClass>org.optaplanner.examples.vehiclerouting.domain.timewindowed.TimeWindowedCustomer</entityClass>
```

Comment:
Definitely needed here because customers are shadow entities. If not declared here, their shadow variable won't be updated.

On the other hand, the constraint verifier doesn't need the entity classes. Apparently, entity classes, or more precisely their descriptors (even for genuine planning entities) are not necessary in order to build a score director used by the constraint verifier to test the provider.

Not sure what to do about that.

Of course, I agree with the inconsistency issue and I can easily fix that by adding those classes. I'm a bit worried though that we're requiring in the verifier build method something that is not used. The same "mistake" can be repeated by users. They don't pass the shadow entity classes (or not even the genuine entity) and--nothing happens.